include/sys/syscall_lookup.h

```diff
@@ -380,3 +380,34 @@ SYSCALL_LOOKUP(munmap,                     2)
 #ifdef CONFIG_CRYPTO_RANDOM_POOL
   SYSCALL_LOOKUP(arc4random_buf,           2)
 #endif
+
+SYSCALL_LOOKUP(exit,                       1)
+SYSCALL_LOOKUP(getrandom,                  3)
+SYSCALL_LOOKUP(nanosleep,                  2)
+
+/* I/O event notification facility */
+
+SYSCALL_LOOKUP(epoll_create1,              1)
+SYSCALL_LOOKUP(epoll_ctl,                  4)
+SYSCALL_LOOKUP(epoll_wait,                 4)
+
+/* Identity of Group/User */
+
+SYSCALL_LOOKUP(getegid,                    1)
+SYSCALL_LOOKUP(geteuid,                    1)
+SYSCALL_LOOKUP(setegid,                    1)
+SYSCALL_LOOKUP(seteuid,                    1)
+
+/* POSIX timers */
+
+SYSCALL_LOOKUP(time,                       1)
+SYSCALL_LOOKUP(gettimeofday,               2)
+SYSCALL_LOOKUP(settimeofday,               2)
+
+/* ANSI C signal handling */
+
+SYSCALL_LOOKUP(signal,                     2)
+
+#ifdef CONFIG_SIGNAL_FD
+  SYSCALL_LOOKUP(signalfd,                 3)
```

Comment:
Let's group this with timerfd and eventfd that are grouped around the poll

Reply:
Done!